.github/workflows/tuist.yml

```diff
@@ -60,9 +60,19 @@ jobs:
       - uses: actions/checkout@v3
       - name: Select Xcode
         run: sudo xcode-select -switch /Applications/Xcode_$(cat .xcode-version).app
+      - uses: asdf-vm/actions/install@v2
+      - name: Select Xcode
+        run: sudo xcode-select -switch /Applications/Xcode_$(cat .xcode-version).app
+      - uses: actions/cache@v3
+        name: 'Cache fetched dependencies folder'
+        with:
+          path: Tuist/Dependencies/SwiftPackageManager/.build
+          key: spm-v1-${{ hashFiles('Package.resolved') }}
+          restore-keys: spm-v1-${{ hashFiles('Package.resolved') }}
+      - name: Fetch dependencies
+        run: tuist fetch
       - name: Test
-        run: |
-          ARGS="--skip-test-targets TuistBuildAcceptanceTests TuistGenerateOneAcceptanceTests" make tuist/test
+        run: tuist test --no-cache --skip-test-targets TuistBuildAcceptanceTests TuistGenerateOneAcceptanceTests
```

Comment:
Why `--no-cache`?

Reply:
It's currently broken for integration tests because when `ProjectDescription` is cached, we need to pass the path to it. We already do that for acceptance tests but we'll need a slightly different solution here as we run these through `Tuist-Workspace` if I'm not mistaken. Ideally, we'd add the env variable to any autogenerated scheme but we currently don't support that.